bool bDetect
int8 nStatus
RectInfo[] vecRectArray
